docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def select_candidates(subsequences, min_dist):
</pre><p>There is one major disadvantage in using subsequences here. For much longer time series, this will be HUGE and use a lot of memory to store (since it will be a 2D array and not a view due to z-normalization)! This is why the original paper probably used the raw time series as input instead. We should probably consider changing this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Correct! I haven't tested it on VERY LONG time series. 

**NOTE:**
In `fastMERLIN`, the `for-loop` in the function `_select_candidates` is iterating through blocks of subsequences (to get advantage from vectorization). In that case, z-normalizing the subsequences of each block shouldn't be a problem. Correct? 
We might follow a similar approach in `MERLIN`. We are not going to change the `for-loop`. However, we can z-normalize the next 256 subsequences in advance. 

What do you think?

Reply:
For this PR, let's avoid any major optimizations that deviate significantly from the pseudocode in the paper. This first iteration should try to stay as close to the original as possible and just have an implementation that works reasonably well.

Reply:
Frankly, if there are already big gaps between the subsequences being used in the `np.matmul` then I'm not sure we benefit that much from vectorization. I would leave vectorization to NumPy for now

Reply:
Ok. So, for now, I leave it as is. We can get back to it later to tackle this  challenge. I am going to make a short note in the notebook regarding this matter so we have that in mind later. Sounds good?

Reply:
<p>Great!</p>

Reply:
As discussed, I am going to keep it as is (for now) and will make a short note of it in the notebook.